exercises/hangman/README.md

```diff
@@ -0,0 +1 @@
+# Hangman
```

Comment:
You can adopt `README.md` from here - https://github.com/exercism/problem-specifications/blob/master/exercises/hangman/description.md